envoy.dependency.check/envoy/dependency/check/abstract/checker.py

```diff
@@ -96,9 +97,9 @@ def dependency_metadata(self) -> typing.DependenciesDict:
     def disabled_checks(self):
         disabled = {}
         if not self.access_token:
-            disabled["release_dates"] = "No Github access token supplied"
-            disabled["release_issues"] = "No Github access token supplied"
-            disabled["releases"] = "No Github access token supplied"
+            disabled["release_dates"] = _NO_GITHUB_TOKEN_ERROR_MSG
```

Comment:
no need to prefix these messages with `_` i dont think they are particularly private etc